components/aabb-collider/index.js

```diff
@@ -61,6 +61,19 @@ AFRAME.registerComponent('aabb-collider', {
     this.observer.disconnect();
     this.el.sceneEl.removeEventListener('object3dset', this.setDirty);
     this.el.sceneEl.removeEventListener('object3dremove', this.setDirty);
+    if (this.data.debug) {
+      if (this.boxHelper) {
+        this.el.sceneEl.object3D.remove(this.boxHelper);
+        this.boxHelper = null;
```

Comment:
We should also call `this.boxHelper.dispose()` here to remove the geometry and material that was created by `BoxHelper`, see https://github.com/mrdoob/three.js/blob/b07dab28897aff83efba7c185d22a3d3a387fba1/src/helpers/BoxHelper.js#L103-L108

Reply:
actually call dispose before `this.boxHelper = null;` ah ah, so
```
this.boxHelper.dispose && this.boxHelper.dispose();
this.boxHelper = null;
```